channels/ON-CHAIN.md

```diff
@@ -96,6 +96,12 @@ the transaction might end up being stuck in the mempool for an extended period.
 The `fee` and `nonce` refer to the `initiator` account, i.e. the `fee` MUST be
 taken from their balance and the `nonce` of their account MUST be incremented.
 
+#### Generalized accounts
+
+If either of the participants is a [Generalized
+account](../generalized_accounts/generalized_accounts.md) the channel create
+will also create an extra entry in the contract state tree, containing a frozen
+authentication state that will be used for off-chain signing and verification.
```

Comment:
Well, anywhere there is an account it could be a generalized account, but unless that account is signing there is no change. And if it is signing that Tx has to be wrapped in a MetaTx so there isn't any changes to the existing transactions!? Not sure how to convey that message best? 🤔

Reply:
PR https://github.com/aeternity/aeternity/pull/2316 introduced new serialization for:
* State channel force progress tx
* State channel offchain tx

I suggest after updates ^^^ are reflected in this protocol repo, this PR gets rebased on master and details on what-can-include-what-at-which-version as it is getting a bit complex.

Structured or comment is ok.

Reply:
This is now rebased on documentation of (^^)